public/pages/ThreatIntel/components/ThreatIntelAlertTriggerForm/ThreatIntelAlertTriggerForm.tsx

```diff
@@ -0,0 +1,174 @@
+          ...trigger.actions[0],
+          message_template: {
+            ...trigger.actions[0].message_template,
+            source: message,
+          },
+        },
+      ],
+    });
+  };
+
+  const prepareMessage = () => {};
+
+  return (
+    <EuiAccordion
+      id="threat-intel-trigger"
+      initialIsOpen={true}
+      buttonContent={trigger.name}
+      extraAction={<EuiButtonIcon iconType={'trash'} onClick={onDeleteTrgger} />}
+      paddingSize="l"
+    >
+      <EuiFormRow>
+        <EuiFieldText
+          placeholder="Trigger name"
+          value={trigger.name}
+          onChange={(event) => {
+            updateTrigger({
+              ...trigger,
+              name: event.target.value,
+            });
+          }}
+        />
+      </EuiFormRow>
+      <EuiSpacer />
+      <EuiAccordion
+        id={'threat-intel-trigger-condition'}
+        buttonContent="Trigger condition"
+        initialIsOpen={true}
+        paddingSize="l"
+      >
+        <EuiFormRow label="Indicator type(s)">
+          <EuiComboBox
+            placeholder="Any"
+            options={enabledIocTypes.map((ioc) => ({ label: ioc }))}
+            selectedOptions={trigger.ioc_types.map((iocType) => ({ label: iocType }))}
+            onChange={(options) => {
+              updateTrigger({
+                ...trigger,
+                ioc_types: options.map(({ label }) => label),
+              });
+            }}
+          />
+        </EuiFormRow>
+        <EuiSpacer />
+        <EuiFormRow label="Log source(s)">
+          <EuiComboBox
+            placeholder="Any"
+            options={logSources.map((logSource) => ({ label: logSource }))}
+            selectedOptions={trigger.data_sources.map((source) => ({ label: source }))}
+            onChange={(options) => {
+              updateTrigger({
+                ...trigger,
+                data_sources: options.map(({ label }) => label),
+              });
+            }}
+          />
+        </EuiFormRow>
+      </EuiAccordion>
+      <EuiSpacer />
+      <EuiFormRow label="Alert severity">
+        <EuiComboBox
+          singleSelection
+          options={Object.values(ALERT_SEVERITY_OPTIONS)}
+          selectedOptions={[
+            parseAlertSeverityToOption(trigger.severity) ?? ALERT_SEVERITY_OPTIONS.HIGHEST,
+          ]}
+          onChange={(options) => {
+            updateTrigger({
+              ...trigger,
+              severity: (options[0]?.value ||
+                ALERT_SEVERITY_OPTIONS.HIGHEST.value) as AlertSeverity,
+            });
+          }}
+        />
+      </EuiFormRow>
+      <EuiSpacer />
+      <NotificationForm
+        action={trigger.actions[0]}
+        allNotificationChannels={allNotificationChannels}
+        loadingNotifications={loadingNotifications}
+        onChannelsChange={onChannelsChange}
+        onMessageBodyChange={onMessageBodyChange}
+        onMessageSubjectChange={onMessageSubjectChange}
+        prepareMessage={prepareMessage}
```

Comment:
will we be having notificationTemplate for the message, otherwise we can remove it for now

Reply:
or add a TODO above prepareMessage function

Reply:
Yes, we will have a template, adding in the follow up PR